webrtc-stats.html

```diff
@@ -1500,6 +1500,7 @@
              RTCIceCandidateType      candidateType;
              long                     priority;
              DOMString                url;
+             DOMString                turnUrlProtocol;
```

Comment:
I wonder if the name will cause confusion with the protocol field above. I do not have a strong opinion.